packages/ui/src/machines/authenticator/index.ts

```diff
@@ -178,6 +178,24 @@ export function createAuthenticatorMachine() {
           },
         },
         authenticated: {
+          initial: 'idle',
+          states: {
+            idle: {
+              on: {
+                TOKEN_REFRESH: 'refreshUser',
+              },
+            },
+            refreshUser: {
```

Comment:
It will not, they will be caught under https://github.com/aws-amplify/amplify-ui/blob/4c6ca0abd73a6b7df51193c27223ac2befb43c9a/packages/ui/src/helpers/authenticator/facade.ts#L74-L75